src/CoreBundle/private/rendering/objectviews/TCMSFields/TCMSFieldBlob/postload.view.php

```diff
@@ -10,7 +10,7 @@
 ?>
     if ($this->sqlData['<?= $sFieldDatabaseName; ?>'] === serialize(false)) {
         $this->sqlData['<?= $sFieldDatabaseName; ?>'] = false;  // special case - false was serialized
-    } else {
+    } elseif(is_string($this->sqlData['<?= $sFieldDatabaseName; ?>'])) {
```

Comment:
What else will it be?

What prior call is it that sometimes (?) decodes it before?

Reply:
The Problem are the lines below it: `$this->sqlData[$sFieldDatabaseName]` is overridden by it's unserialized value. This means that after calling `PostLoadHook` once, the field in `sqlData` may be filled with an array instead of a string. When `PostLoadHook` is called again `unserialize` fails in PHP8 (Warning in PHP7).

I am unsure why this method is called twice, but adding a type-check to ensure that `unserialize` is always passed a string fixed it.